srcpkgs/spago/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'spago'
+pkgname=spago
+version=0.16.0
+revision=1
+hostmakedepends="
+	curl
+	ncurses-libtinfo-libs
+	stack
+	tar
+	xz
+	zlib
```

Comment:
Are the binaries provided by `tar`, `xz` and `zlib` really necessary during build?

Reply:
If the makefile or whatever calls `tar` for files with those compressions, it would make sense.

Reply:
I'm not familiar with `stack` builds and can't verify the need, but the author needs to be able to justify these build-time dependencies.

Reply:
> Are the binaries provided by `tar`, `xz` and `zlib` really necessary during build?

Stack requires tar and xz and spago requires zlib as a dependency